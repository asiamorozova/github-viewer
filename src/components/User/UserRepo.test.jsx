import React from 'react';
import { shallow } from 'enzyme';
import UserRepo from './UserRepo';

describe('Repo component', () => {
  it('renders Repo', () => {
    const wrapper = shallow(<UserRepo 
      name="asiamorozova"
      html_url="https://github.com/asiamorozova"/>);
    expect(wrapper).toMatchSnapshot();
  });
});
