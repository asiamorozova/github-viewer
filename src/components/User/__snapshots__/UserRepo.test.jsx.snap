// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Repo component renders Repo 1`] = `
<li>
  <p>
    asiamorozova
  </p>
  <p>
    https://github.com/asiamorozova
  </p>
</li>
`;
